scripts/gha/lint_commenter.py

```diff
@@ -37,15 +37,18 @@
 import urllib.parse
 
 # Put any lint warnings you want to fully ignore into this list.
+# (Include a short explanation of why.)
 IGNORE_LINT_WARNINGS = [
-  'build/include_subdir',
-  'readability/casting',
-  'whitespace/indent',
-  'whitespace/line_length'
+  'build/include_subdir',   # doesn't know about our include paths
+  'build/c++11',            # ignore "unapproved c++11 header" warning
```

Comment:
Where can I see what warnings fall into this category? I am wondering if this might cause us to miss out on some other useful lint warnings.

Reply:
You can search this file for "build/c++11": https://github.com/google/styleguide/blob/gh-pages/cpplint/cpplint.py#L5854

It appears to only have the "unapproved C++11 header" and "unapproved C++11 class or function" - since we aren't writing code on the internal repo, I think it's probably fine.

Reply:
Ok got it. The check seems to be just for unapproved headers and unapproved classes, and the reason for such things to be unapproved is generally because there is a google3 version of the same concept, but we are not in the google3 codebase. So disabling the check looks good to me.